bin/cli.js

```diff
@@ -15,23 +15,45 @@ const yargsConfiguration = (yargs) => {
 }
 
 const processArgs = async (args) => {
-  const data = await db.JSON();
-  if (args.action == 'add') {
-    const { todos = [] } = data;
-    const nextIndex = todos.length;
-    console.log(`adding ${nextIndex}:${args.actionArg} in todos.`);
-    todos.push(args.actionArg);
-    db.JSON({ todos });
-    await db.sync();
-  } else {
-    console.log('TODOS:');
-    const { todos = [] } = data;
-    todos.forEach((todo, i) => {
-      console.log(`${i}: ${todo}`);
-    });
-    if (!todos.length) {
-      console.log('no todos yet');
-    }
+  switch (args.action) {
+    /**
+     * add a todo item in todos list
+     */
+    case 'add':
```

Comment:
We should try splitting all these cases into multiple functions, each with their own responsibility so it's easier to manage inputs to each case.